.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,2 +1,11 @@
-# This file cannot use the extension `.yaml`.
-blank_issues_enabled: false
+blank_issues_enabled: true
+contact_links:
+  - name: Question
+    url: https://github.com/astral-sh/ruff/discussions
+    about: Ask questions in GitHub Discussions.
```

Comment:
I'd prefer to keep using issues for questions. Let's decide this separtely from this PR

Why: because they sometimes turn into bug reports which then requires transferring issues (which creates noise because it results in extra notifications). I also sometimes go over existing issues to see if they've been resolved, and I'd prefer if I don't have to do this for discussions and issues.

Reply:
That's reasonable. I'll revert it.